proto/p4/v1/p4runtime.proto

```diff
@@ -451,6 +451,8 @@ message Replica {
 message MulticastGroupEntry {
   uint32 multicast_group_id = 1;
   repeated Replica replicas = 2;
+  // Arbitrary metadata from the controller that is opaque to the target.
```

Comment:
There should be a comment about which version of the P4Runtime API specification in which this field was added.  See https://github.com/p4lang/p4runtime/issues/441

Reply:
On 9/8/2023 we agreed to incrementally add to the revision history in the main .mdk spec.